include/tins/hw_address.h

```diff
@@ -131,7 +131,7 @@ class HWAddress {
      * 
      * \param ptr The pointer from which to construct this address.
      */
-    HWAddress(const storage_type* ptr = 0) {
+    HWAddress(const storage_type* ptr = nullptr) {
```

Comment:
This breaks in C++ < 11. We probably should drop support for C++03 and enforce people to use C++11 but until then these sort of changes can't be made or must be guarded

Reply:
Right, no problem.
I'll just carry the patch in our repo.